images/kubeflow-katib/tests/smoke-test.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+# Ref: https://www.kubeflow.org/docs/components/katib/hyperparameter/#installing-katib
+
+NAMESPACE=kubeflow
+KATIB_VERSION=0.15.0
+
+
+function manifests() {
+  # Get the directory path of this script
+  SCRIPT_DIR=$(
+  	cd "$(dirname "$0")"
+  	pwd
+  )
+
+
+  envsubst < $SCRIPT_DIR/kubeflow-katib.tpl.yaml > $SCRIPT_DIR/kubeflow-katib.yaml
+  cat $SCRIPT_DIR/kubeflow-katib.yaml
+  cat <<EOF > kustomization.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - github.com/kubeflow/katib.git/manifests/v1beta1/installs/katib-standalone?ref=v${KATIB_VERSION}
+images:
+  - name: docker.io/kubeflowkatib/katib-controller
+    newName: ${IMAGE_REPOSITORY_CONTROLLER}
+    newTag: ${IMAGE_REPOSITORY_CONTROLLER_TAG}
+  - name: docker.io/kubeflowkatib/katib-db-manager
+    newName: ${IMAGE_REPOSITORY_DB_MANAGER}
+    newTag: ${IMAGE_REPOSITORY_DB_MANAGER_TAG}
+namespace: ${NAMESPACE}
+EOF
+}
+
+manifests
+kubectl apply -k .
+
+kubectl rollout status -n kubeflow deployment/katib-db-manager --timeout=120s -n kubeflow 
+kubectl rollout status -n kubeflow deployment/katib-controller --timeout=120s -n kubeflow 
```

Comment:
Namespace is stated twice here:
```suggestion
kubectl rollout status -n ${NAMESPACE} deployment/katib-db-manager --timeout=120s 
kubectl rollout status -n ${NAMESPACE} deployment/katib-controller --timeout=120s 
```